e2e/startupSmoke.spec.ts

```diff
@@ -18,17 +18,41 @@ class DiveInfoPage {
         return this.page.locator(this.waypointRowsSelector);
     }
 }
+class SacCalculatorPage {
+    private readonly calculatorsMenuSelector = '#calculators-menu';
+    private readonly rmvSacOptionSelector = 'a[routerLink="/sac"]';
+    private readonly diveTimeInputSelector = '#dive-time-input';
+    private readonly rmvValueSelector = '#total-rmv-valuesac';
 
-test.describe('Planner shows calculated profile', () => {
-    let diveInfoPage: DiveInfoPage;
+    constructor(private page: Page) {}
+
+    async navigateToRmvSacCalculator() {
+        await this.page.goto('/');
+        await this.page.locator(this.calculatorsMenuSelector).click();
+        await this.page.locator(this.rmvSacOptionSelector).click();
```

Comment:
You dont have to go through the menu, just directly navigate to the target page.
Then you also dont need the menu ids.